### PR TITLE
Tutorial layout validation

### PR DESCRIPTION
**Pull request by bencbernstein**
If we do a significant upgrade to the tutorial I'll refactor this… consequence of designing and coding at the same time.

### CODE REVIEW

File: Apiary/ViewControllers/TutorialViewController.swift [hunk 1]
```diff
@@ -9,21 +9,35 @@ extension Notification.Name {
   static let advanceInTutorial = Notification.Name("advance-in-tutorial")
 }
 
+enum TutorialView {
```
Comment (OLI9292) [inline Apiary/ViewControllers/TutorialViewController.swift]: any reason for an enum? an Array supports iteration and is simpler

File: Apiary/Views/TutorialViews/TutorialQuestionThree.swift [hunk 2]
```diff
@@ -79,20 +84,58 @@ class TutorialQuestionThree: UIView {
...
     
     delay(seconds: 3.5) {
+      animate(string: self.questionText, label: self.questionLabel)
+    }
+    
+    delay(seconds: 5.5) {
       UIView.animate(withDuration: 0.2, animations: {
-        self.buttons.forEach({ (button) in
-          button.alpha = 1
+        self.questionComponents.forEach({ (questionComponent) in
```
Comment (OLI9292) [inline Apiary/Views/TutorialViews/TutorialQuestionThree.swift]: `self.questionComponents.forEach { $0.alpha = 1 }` is a little cleaner # minor

File: Apiary/ViewControllers/TutorialViewController.swift (hunk 1 above)
Comment (bencbernstein) [inline Apiary/ViewControllers/TutorialViewController.swift]: we can't hold an array of instantiated views or the animation won't trigger properly

Comment (OLI9292) [inline Apiary/ViewControllers/TutorialViewController.swift]: what doesn't trigger?

**Pull request merged**